go.mod

```diff
@@ -189,3 +189,5 @@ replace (
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.2-alpha.regen.4
 	google.golang.org/grpc => google.golang.org/grpc v1.33.2
 )
+
+replace github.com/rollkit/celestia-openrpc => ../celestia-openrpc
```

Comment:
Remove once https://github.com/rollkit/celestia-openrpc/pull/43 has merged.

Reply:
@Wondertan https://github.com/rollkit/celestia-openrpc/pull/43 is merged

Reply:
Is it released?